services/profile.ts

```diff
@@ -77,7 +77,11 @@ const invalidSignatureErrorHandler = (error: FetchError) => {
 }
 
 export const toSubstrateAddress = (address: string) =>
-  isEthereumAddress(address) ? address : formatAddress(address, 42)
+  !address
```

Comment:
we should change the name of this function, is not really `toSubstrateAddress` anymore , now that we've added evm

Reply:
I would leave it the same as the naming change could cause code conflict between branchs.